plugins/stingray/the_debuginator_plugin/c_api_the_debuginator.cpp

```diff
@@ -376,6 +383,35 @@ void handle_default_input(TheDebuginator* debuginator, unsigned devices) {
 					}
 				}
 
+	if (devices & Debuginator_Mouse) {
+		CApiInputControllerPtr mouse = script_api->Input->mouse();
+		input_wrapper.controller = mouse;
+
+		if (api->any_released(input_wrapper.controller) != UINT_MAX) {
+			input_wrapper.time_since_pressed = -1;
+		}
+
+
+		if (debuginator_is_open(debuginator)) {
+			CApiVector3 scroll = input_wrapper.axis("wheel");
+			if (scroll.y != 0) {
+				debuginator->focus_height += scroll.y * 0.05f;
```

Comment:
Right, ok that makes sense. :)

If it's ok with you, I'll accept this after you fix the other issues, but there are some things I think I would like to change with how this is done. I want to move some of the functionality to The Debuginator. I don't want each app to have to reinvent how to do mouse-input.